### ISSUE TITLE
SVG tags get incorrect CSS class

### ISSUE THREAD

**Issue by PierBover** [imba/imba#201]
So when declaring a tag based on SVG the default CSS class gets a underscore at the beginning.

So for my `Spinner` tag the class becomes `_Spinner`.

**Comment by PierBover** [comment]
Apparently here is the culprit:

https://github.com/imba/imba/blob/master/lib/imba/dom/svg.js#L24

Is there any reason for this @somebee ? Would you accept a PR to change this?

**Comment by somebee** [comment]
Definitely - this is a bug :/

**Comment by scanf** [comment]
@PierBover are you planning on sending a pull request for this?

**Comment by PierBover** [comment]
@scanf yes, but I'm in the midst of deadlines so I don't know when. The fix seems pretty easy although I never got around to testing it properly.

**Comment by scanf** [comment]
Thanks for letting me know. Feel free to follow up on this when you get time. I am interested in the SVG bits but there is plenty of other topics I can look at first. 

Happy coding.

**Comment by taw** [comment]
I created PR as I ran into this issue a lot too.
